src/consolebundle/CommandManager.py

```diff
@@ -9,9 +9,40 @@ def __init__(self, commands: List[ConsoleCommand]):
     def get_commands(self):
         return self.__commands
 
-    def get_by_name(self, name: str) -> ConsoleCommand:
+    def log_commands(self, logger):
+        print("\n[Available commands]:", flush=True)
+        for existing_command in self.get_commands():
+            logger.info(existing_command.get_command() + " - " + existing_command.get_description())
+
+    def log_subcommands(self, command_name: list, logger):
+        print("\n[Available space-separated commands]:", flush=True)
+        command_prefix = ':'.join(command_name)
+        for existing_command in self.get_commands():
+            if existing_command.get_command().startswith(command_prefix):
+                logger.info(
+                    f"{' '.join(existing_command.get_command().split(':'))} - " + existing_command.get_description())
+
+    def get_by_name(self, name: list) -> ConsoleCommand:
```

Comment:
Why list?

Reply:
Name is now a list. It is because of the way argparse works. It does not allow optional positional arguments and command_name is a positional argument. 
So if we want to allow both dbx:deploy and dbx deploy commands to be valid, we have to do it like this. So command_name is either ['dbx:deploy'] or ['dbx', 'deploy'] in this case.

Reply:
Ok, so let's rename it to name_parts or something like that. name: list is confusing to me

Reply:
Already done.